lineman/app/components/profile_page/profile_page_controller.coffee

```diff
@@ -1,17 +1,35 @@
-angular.module('loomioApp').controller 'ProfilePageController', ($rootScope, CurrentUser, Records, FlashService, $location) ->
-  @user = CurrentUser.clone()
+angular.module('loomioApp').controller 'ProfilePageController', ($rootScope, Records, FlashService, $location, AbilityService, ModalService, ChangePictureForm, ChangePasswordForm, DeactivateUserForm) ->
+  @init = ->
+    @user = Records.users.find(window.Loomio.currentUserId)
```

Comment:
Random thought - Maybe it's a problem with correctly updating the indexes in lokijs (I'm pretty sure we're not calling update() on loki when we update a record, which tells it to reindex). Are there any loki views involved?